src/main/java/net/sf/jsqlparser/statement/select/Distinct.java

```diff
@@ -43,9 +44,18 @@ public void setUseUnique(boolean useUnique) {
         this.useUnique = useUnique;
     }
 
+    public boolean isUseDistinctRow() {
```

Comment:
Since [`DISTINCTROW` is a synonym for `DISTINCT`](https://dev.mysql.com/doc/refman/8.4/en/select.html), I re-used the existing `Distinct` class and only augmented it with the `useDistinctRow` attribute to determine the string representation of `Distinct`.